tests/components/Sidebar-cy.js

```diff
@@ -142,31 +142,30 @@ describe("Sidebar", function() {
       });
     });
 
-    it("renders within the viewport on initial load", function() {
-      cy.get(".sidebar").then(function($sidebar) {
-        const sidebarRect = $sidebar.get(0).getBoundingClientRect();
+    it("Close sidebar when sidebarToggle button clicked", function() {
+      cy.get(".page-header-sidebar-toggle").click();
+      cy.get(".sidebar-visible").should("not.exist");
+    });
 
-        // Assert that the sidebar's dimensions are within the viewport.
-        expect(sidebarRect.left).to.equal(0);
-        expect(sidebarRect.top).to.equal(0);
-        expect(sidebarRect.right).to.be.greaterThan(0);
+    it("Automatically close sidebar when view is mobile/tablet", function() {
+      cy.viewport("iphone-6");
+      cy.get(".sidebar-visible.sidebar-docked").should("not.exist");
 
-        cy.window().then(function($window) {
-          expect(sidebarRect.bottom).to.equal($window.innerHeight);
-        });
-      });
+      // reset viewport back
+      cy.viewport("macbook-15");
     });
 
-    it("offsets the page content so that they are sitting directly adjacent to one another without overlapping", function() {
-      cy.get(".sidebar").then(function($sidebar) {
-        const sidebarRect = $sidebar.get(0).getBoundingClientRect();
+    it("Display overlay when sidebar is open on mobile/tablet", function() {
```

Comment:
Can we test toggle for mobile too

Reply:
mobile doesn't change the environment.. the toggle is the same so I believe there is no need to test the button again on mobile.